### PR TITLE
Chore/auth updates updates/mfranz1

### PR DESCRIPTION
**Pull request by mfranz1**

### CODE REVIEW

File: src/app/cube/home/home.component.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
   ></clark-blogs>
   <!-- remove after clark 5th birthday -->
   <div style="display: flex; flex-direction: row; justify-content: center;">
-    <h1 style="background: none;text-align: center; position: absolute; z-index: 10; ">HAPPY 5TH BIRTHDAY CLARK</h1>
+    <h1 style="background: none;text-align: center; position: absolute; z-index: 10; ">HAPPY 5TH BIRTHDAY <span style="color: #1C70DD">CLARK</span></h1>
```
Comment (mfranz1) [inline src/app/cube/home/home.component.html:9]: The banner will be removed so this hard coded color is fine.

**Review by pmanal1 — APPROVED** [review]

**Review by jouypo23 — APPROVED** [review]

**Review by JosueMolinaMorales — APPROVED** [review]

**Review by EVogel1999 — APPROVED** [review]

File: src/app/cube/home/home.component.html (hunk 1 above)
Comment (katieblomquist) [reply src/app/cube/home/home.component.html:9]: Maybe use the new homepage blue here? #376ED6

File: src/app/cube/home/splash/splash.component.scss [hunk 2]
```diff
@@ -7,10 +7,7 @@
     background: linear-gradient(0deg, $primary-white 0%, rgba(255, 255, 255, 0) 100%);
     border: 1px solid $secondary-white;
     border-top: none;
-
-    //remove after 5th birthday
-    margin-top: 100px;
-    //remove after 5th birthday
+    padding-top: 135px;
```
Comment (katieblomquist) [inline src/app/cube/home/splash/splash.component.scss:10]: I'd add a "remove after 5th birthday comment" so this doesn't get missed when we remove the banner.

**Review by katieblomquist — COMMENTED** [review]
Just the 2 small things, but other than that, this looks good to me!

**Review by katieblomquist — APPROVED** [review]

**Pull request merged**